indico/htdocs/sass/partials/_inputs.scss

```diff
@@ -725,6 +750,38 @@ $dropdown-transition: $dropdown-transition-step ease-out;
     }
 }
 
+.i-has-action.i-synced-input {
+    > input[type=checkbox] {
+        position: absolute;
+        top: 0;
+        left: 0;
+        display: none;
+        opacity: 0;
+    }
+    > label.i-synced-label {
+        text-align: center;
+        line-height: 2.3em;
+    }
+
+    > input[type=checkbox]:not(:checked) ~ .i-synced-label:before { color: $light-black; }
+    > input[type=checkbox]:checked ~ .i-synced-label:before { color: $blue; }
+
+    > input[type=text], textarea {
+        &[readonly] {
+            background-color: lighten($pastel-gray, $light-color-variation);
+            background: repeating-linear-gradient(
+                -40deg,
+                lighten($pastel-gray, 0.5*$color-variation),
```

Comment:
What about these magic numbers? They kind of defeat the purpose of the palette and `$color-variation`. Can't we find any combination of not magic colors that plays well on the interface?